lisp/mastodon-tl.el

```diff
@@ -244,9 +244,10 @@ Optionally start from POS."
                  'face 'mastodon-handle-face
                  'mouse-face 'highlight
 		 ;; TODO: Replace url browsing with native profile viewing
-		 'mastodon-tab-stop 'shr-url
+		 'mastodon-tab-stop 'handle
 		 'shr-url profile-url
-		 'keymap mastodon-tl--shr-map-replacement
+		 'keymap mastodon-tl--link-keymap
+                 'mastodon-handle (concat "@" handle)
```

Comment:
Here we actually have the account info, why not store that in the property (or some other property)?
Seems wasteful (resource wasting for the server and time wasting for the user) to go back to the server to look up information we already have.

Reply:
Good call. Since the account is in the toot, we can check if the handle has 'toot-json before doing a search and extract the account from there.